src/include/daos_srv/evtree.h

```diff
@@ -65,6 +64,11 @@ struct evt_desc {
 	uint32_t			dc_magic;
 	/** The DTX entry in SCM. */
 	umem_id_t			dc_dtx;
+	/** number of csums stored in pt_csum array */
+	uint32_t			pt_csum_count;
```

Comment:
can this be calculated simply by knowing the size of the extent?

Reply:
Yes, probably. I'll make that change.